presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -313,7 +313,13 @@ Property Name                                           Description
 
                                                         Set to ``0`` to disable metadata optimization.
 
-``iceberg.split-manager-threads``                       Number of threads to use for generating Iceberg splits.        ``Number of available processors``
+``iceberg.split-manager-threads``                       Number of threads to use for generating Iceberg splits.       ``Number of available processors``
+
+``iceberg.metadata-previous-versions-max``              The max number of previous metadata files recorded in         ``100``
+                                                        current metadata.
+
+``iceberg.metadata-delete-after-commit``                Whether enables to delete the oldest metadata files after      ``false``
```

Comment:
```suggestion
``iceberg.metadata-delete-after-commit``                Set to ``true`` to delete the oldest metadata files after      ``false``
```
This is broken across a couple of lines because of the table columns formatting, but I'm suggesting to change "Whether enables to delete the oldest metadata files after each commit" to "Set to ``true`` to delete the oldest metadata files after each commit."

Reply:
Thanks for the suggestion, fixed!